polling/assets.go

```diff
@@ -346,23 +346,45 @@ func (d *DataSources) PullBestPrice(asset string, reference time.Time, sources m
 
 		if price.Value != 0 {
 			prices = append(prices, price)
+		}
+	}
 
-			// We can break out if this is a 'good' price
-			//	Is it stale AND the market is open? We can expect stale quotes in a closed market
-			if reference.Sub(price.When) > d.staleDuration && IsMarketOpen(asset, reference) {
-				// This price quote is stale, keep fetching prices
-				continue
+	if oprversion == 5 {
+		pricesClone := prices
```

Comment:
why cloned version?

Reply:
We should keep the original priority in the prices list.